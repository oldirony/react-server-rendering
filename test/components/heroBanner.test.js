import React from 'react';
import HeroBanner from '../../src/components/hero-banner/hero-banner';
import {shallow} from 'enzyme';

test('Base rendering', ()=>{
	const hero = shallow(<HeroBanner />);
	expect(hero).toMatchSnapshot();
});