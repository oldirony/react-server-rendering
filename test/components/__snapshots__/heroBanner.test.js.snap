// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base rendering 1`] = `
<div
  className="banner"
>
  <h1
    className="title"
  >
    Ideas!
  </h1>
</div>
`;
